__tests__/app.test.js

```diff
@@ -0,0 +1,40 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data/index.js");
+
+afterAll(() => {
+  if (db.end) db.end();
+});
+beforeEach(() => {
+  return seed(testData);
+});
+
+describe("GET api/topics", () => {
+  test('"status:200, responds with an array of topic objects" ', () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        const { topics } = body;
+        expect(topics).toBeInstanceOf(Array);
+        topics.forEach((topic) => {
+          expect(topic).toEqual(
+            expect.objectContaining({
+              slug: expect.any(String),
+              description: expect.any(String),
+            })
+          );
+        });
+      });
+  });
+  test("status:404, responds with error message when passed an invalid route", () => {
```

Comment:
lovely test once again -  I would potentially make this 404 it's own describe. This is because a bad path or invalid route isn't specific to 'topics', it's a whole-API kind of error. I'd maybe make it's own describe block outside of GET api/topics and put it there